lib/parse-styles.js

```diff
@@ -93,6 +94,10 @@ async function resolveImportId(result, stmt, options, state, postcss) {
     return
   }
 
+  if (dataURL.isValid(stmt.from)) {
+    return
+  }
```

Comment:
Perhaps leave a comment on what we're doing here, and why? Won't be obvious to our future selves.